tests/datastructures/test_headers.py

```diff
@@ -271,10 +270,10 @@ def test_cache_control_from_header_single_value() -> None:
     header_value = "no-cache"
     header = CacheControlHeader.from_header(header_value)
     header_dict = header.dict(exclude_unset=True, exclude_none=True, by_alias=True)
-    assert header_dict == {"no-cache": True}
+    assert header_dict == {"documentation-only": False, "no-cache": True}
```

Comment:
Why do we expect the `"documentation-only"` directive in the output now? Was the original test incorrect?